From 4f1c2391d00069d5c064c1ea6b29cea998c48022 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 30 May 2025 17:34:44 -0700
Subject: [PATCH 2/2] Remove mapbox

---
 src/plugins/geoservices/geoservices.pro | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/plugins/geoservices/geoservices.pro b/src/plugins/geoservices/geoservices.pro
index b81ad34..bc3a242 100644
--- a/src/plugins/geoservices/geoservices.pro
+++ b/src/plugins/geoservices/geoservices.pro
@@ -3,16 +3,6 @@ TEMPLATE = subdirs
 QT_FOR_CONFIG += location-private  # pulls in the features defined in configure.json
 
 qtConfig(geoservices_here): SUBDIRS += nokia
-qtConfig(geoservices_mapbox): SUBDIRS += mapbox
 qtConfig(geoservices_esri): SUBDIRS += esri
 qtConfig(geoservices_itemsoverlay): SUBDIRS += itemsoverlay
 qtConfig(geoservices_osm): SUBDIRS += osm
-
-qtConfig(geoservices_mapboxgl) {
-    !exists(../../3rdparty/mapbox-gl-native/mapbox-gl-native.pro) {
-        warning("Submodule mapbox-gl-native does not exist. Run 'git submodule update --init' on qtlocation.")
-    } else {
-        SUBDIRS += mapboxgl ../../3rdparty/mapbox-gl-native
-        mapboxgl.depends = ../../3rdparty/mapbox-gl-native
-    }
-}
-- 
2.49.0

